src/frontends/tensorflow_lite/src/schema/schema.fbs

```diff
@@ -0,0 +1,1333 @@
+// Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I suppose we need to add some information about this file to specific file from https://github.com/openvinotoolkit/openvino/tree/master/licensing and here https://github.com/openvinotoolkit/openvino/blob/master/cmake/packaging/copyrights/generic

Please, contact Yury Semaev about how to do it.

Reply:
Contacted. Waiting for approval from Yuri

Reply:
Corrected both licencing and generic files according to Yuri recomendations.